standard/statements.md

```diff
@@ -1809,13 +1889,13 @@ resource_acquisition
     ;
 ```
 
-A ***resource*** is a class or struct that implements the `System.IDisposable` interface, which includes a single parameterless method named `Dispose`. Code that is using a resource can call `Dispose` to indicate that the resource is no longer needed.
+A ***resource*** is a class or struct that implements the `System.IDisposable` (`IAsyncDisposable` for async streams) interface, which includes a single parameterless method named `Dispose` (`DisposeAsync` for async streams). Code that is using a resource can call `Dispose` to indicate that the resource is no longer needed.
```

Comment:
I think all the async stream bits need reworking here and below.

Reply:
Possibly just define "async resources" instead? Although that's a bit weird. I do wonder whether we should drop "resource" as a term entirely, and talk about disposables and asynchronous disposables, akin to enumerators and asynchronous enumerators, and enumerables and async enumerables. Basically name these by what you can do, not any expected meaning.